src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/courtsentencing/CourtSentencingSynchronisationService.kt

```diff
@@ -807,6 +807,40 @@ class CourtSentencingSynchronisationService(
     }
   }
 
+  suspend fun nomisCaseIdentifiersUpdated(eventName: String, event: CaseIdentifiersEvent) {
+    val telemetry =
+      mapOf(
+        "nomisIdentifiersNo" to event.identifierNo,
+        "nomisIdentifiersType" to event.identifierType,
+        "nomisCourtCaseId" to event.caseId.toString(),
+        "offenderNo" to event.offenderIdDisplay,
+        "eventType" to eventName,
+      )
+    if (event.auditModuleName == "DPS_SYNCHRONISATION") {
+      telemetryClient.trackEvent("case-identifiers-synchronisation-skipped", telemetry)
+    } else {
```

Comment:
more filtering to come to ignore case references for identifiers that DPS are not interested in